packages/arm-core/src/components/controls/MapControls.tsx

```diff
@@ -0,0 +1,12 @@
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+const MapControls = styled.div`
+  position: relative;
+  width: 100%;
```

Comment:
`width` Is an implied rule for block-level elements, like divs

Reply:
:ok_hand: